src/language-css/printer-postcss.js

```diff
@@ -52,12 +50,13 @@ import {
 function genericPrint(path, options, print) {
   const { node } = path;
 
-  switch (node.type) {
+  // TODO: Do we want to *just* check for `sassType`?
+  switch (node.type ?? node.sassType) {
```

Comment:
sass-parser adds a new `sassType` field, which is more useful for differentiating node types. We probably want to use that consistently when sass-parser is being used?